docs/intro.rst

```diff
@@ -83,11 +78,31 @@ Hadoop in pseudo-distributed mode, see this
 Installation
 ------------
 
-You can download Confluent Platform at .... And follow the instructions to install from
-rpm, deb or archives.
+.. ifconfig:: platform_docs
+
+   See the :ref:`installation instructions<installation>` for the Confluent
+   Platform. Before starting a Camus job you must have Haddop, Kafka, and the
+   Schema Registry running. The :ref:`Confluent Platform quickstart<quickstart>`
+   explains how to start Kafka and the Schema Registry locally for testing.
+   See this
+   `guide <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cdh_qs_cdh5_pseudo.html>`_.
+   to setup a single Hadoop node in pseudo-distributed mode.
+
+.. ifconfig:: not platform_docs
+
+   You can download prebuilt versions of Camus as part of the
+   `Confluent Platform <http://confluent.io/downloads/>`_. To install from
+   source, follow the instructions in the `Development`_ section.
+   Before starting a Camus job you must have Haddop, Kafka, and the
```

Comment:
This looks great. Let's link to the place that tells users how to start Kafka and Schema Registry. @ewencp Does this sort of cross repo linking have to wait until the docs are published?